ddl/placement/common.go

```diff
@@ -30,19 +30,6 @@ func GroupID(id int64) string {
 	return fmt.Sprintf("%s%d", BundleIDPrefix, id)
 }
 
-const (
```

Comment:
You could create on your own, we don't need to reuse the same definition. It is controlled by two different system.